developer_docs/provisioning_setup.asciidoc

```diff
@@ -0,0 +1,258 @@
+[[api-v2-development]]
+= Provisioning setup
+:toc: right
+:toclevels: 5
+
+Guide how to setup provisioning on Fedora with Libvirt.
+
+[[tftp-configuration]]
+== TFTP configuration
+
+[source, bash]
+....
+dnf install -y syslinux
+mkdir -p /var/lib/tftpboot
+mkdir -p /var/lib/tftpboot/{boot,grub,grub2,pxelinux.cfg}
+cp /usr/share/syslinux/{pxelinux.0,menu.c32,chain.c32,ldlinux.c32,libcom32.c32,libutil.c32} /var/lib/tftpboot
+
+# Change file owner to your account
+chown your-account:your-group -R /var/lib/tftpboot
+....
+
+[[libvirt-configuration]]
+== Libvirt configuration
+Install Libvirt
+[source, bash]
+....
+dnf install @virtualization
+....
+
+Check that `libvirtd` is running
+[source, bash]
+....
+systemctl start libvirtd
+systemctl enable libvirtd
+systemctl status libvirtd
+....
+
+Edit Libvirt `default` network
+[source, bash]
+....
+# virsh net-edit default
+
+<network>
+  <name>default</name>
+  <uuid>your-uuid</uuid>
+  <forward mode='nat'/>
+  <bridge name='virbr0' stp='on' delay='0'/>
+  <mac address='your-mac'/>
+  <domain name='local.loc' localOnly='yes'/>
```

Comment:
I would not invent the `.loc` TLD. At some point someone can buy it and you could create problems. Always use one that's properly defined. https://en.wikipedia.org/wiki/List_of_Internet_top-level_domains#Special-Use_Domains suggests `.test` might be the best.